### PR TITLE
docs: update developer doc

### LINKED ISSUE #814: Update BC Wallet developer setup documents
We would like the developer documentation updated to reflect the current state. If you can, please create a PR agains the BC Wallet repo with updates as needed.

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Resolves: https://github.com/bcgov/bc-wallet-mobile/issues/814

I didn't cover iOS setup / debugging much due to me working on a Windows machine, so further updates from someone who works with a Mac + iPhone would be appreciated

### CODE REVIEW

**Comment by jleach** [comment]
No need for the Signed-off message in the description. We have a bot that checks for it.

**Review by wadeking98 — COMMENTED** [review]
Might want to remove the line about von-network if we're recommending that users use BCovrin

File: DEVELOPER.md [hunk 1]
```diff
@@ -1,106 +1,159 @@
...
-  > **Tip**: It is recommended to install the [Android Studio](https://developer.android.com/studio), it comes with
-  android sdk,
-  > device manager to install emulators.
-* JDK 8 or 11 (Preferably OpenJDK or ZuluJDK)
 
-## Prerequisite Services
+## Prerequisite services
 Requires the following services to be up and running.
 
 - [Von Network](https://github.com/bcgov/von-network)
```
Comment (wadeking98) [inline DEVELOPER.md:26]: Might want to remove the line about von network if we're recommending that users use BCovrin instead

**Review by wadeking98 — APPROVED** [review]
Very thorough documentation, well done

**Pull request merged**